docs/windows_dev.md

```diff
@@ -0,0 +1,99 @@
+# Integration Tests: Windows Local setup
+**Prerequisites**: 
+- You have already setup Go and the buildkit repo and you can build from source.
+
+
+### Main Steps
+- Install registry.exe:
+```go install -v github.com/distribution/distribution/v3/cmd/registry@latest```
+- Install buildkit with the following command: 
+     ```
+    cd to repo directory
+    go install -v .\cmd\...
+    ```
+
+- Run buildkitd from the _$env:PATH_. Also remember to remove any other buildkitd installed in $env:ProgramFiles.
+- In order to run a test we'll focus on the tests written in the _frontend/dockerfile/dockerfile_test.go_ file.
+- Pick a specific test to run, eg. _testEnvEmptyFormatting_. This is how you run it: 
+   > Note the capitalization of the test.
+    ```
+    # cd to the package you want to test, e.g. frontend/dockerfile
+    cd frontend/dockerfile
+    # run a specific test
+    go test -v --timeout=60m --run=TestIntegration/TestEnvEmptyFormatting/worker=containerd --timeout=60m 
```

Comment:
To simplify, just remove the `--timeout` arg, you also added it twice. And then on the note below, can mentioned. We can optimize for folks quickly copy-pasting the commands. So optional ones, can be left out.